learning_resources_search/management/commands/recreate_index.py

```diff
@@ -44,12 +50,24 @@ def handle(self, *args, **options):  # noqa: ARG002
                 for object_type in sorted(ALL_INDEX_TYPES):
                     self.stdout.write(f"  --{object_type}s")
                 return
-
-            task = start_recreate_index.delay(indexes_to_update)
-            self.stdout.write(
-                f"Started celery task {task} to index content for the following"
-                f" indexes: {indexes_to_update}"
+        if not remove_existing_reindexing_tags:
+            existing_reindexing_indexes = get_existing_reindexing_indexes(
+                indexes_to_update
             )
+            if existing_reindexing_indexes:
+                self.stdout.write(
+                    f"Reindexing in progress. Reindexing indexes already exist:"
+                    f" {', '.join(existing_reindexing_indexes)}"
+                )
+                return
```

Comment:
If you leave this new conditional block out, and `remove_existing_reindexing_tags` is False, the user will still be notified of the problem, because any errors that occur when running the task are shown later below:

> mattbert@Matts-MacBook-Pro mit-open % docker compose exec web ./manage.py recreate_index --course
Started celery task 722af285-d989-450f-9eb8-4a9070f31f1a to index content for the following indexes: ['course']
Waiting on task...
CommandError: Recreate index errored: Reindexing in progress. Reindexing indexes already exist: discussions_local_course_67171d570c7e4386afd5b4a957669c2e